src/site/_includes/macros/authorcard.njk

```diff
@@ -0,0 +1,31 @@
+{% macro authorcard(collectionItem) %}
```

Comment:
See comment below about changing this argument from `collectionItem` to `post|tag|author` depending on which card macro we're using.